src/com/csciOOD/Screen.java

```diff
@@ -1,23 +1,99 @@
 package com.csciOOD;
 
 import javax.swing.*;
-import java.awt.event.ActionEvent;
-import java.awt.event.ActionListener;
+import java.awt.*;
 import com.csciOOD.Game;
+import com.csciOOD.Menu;
 
-public class Screen extends JFrame implements ActionListener {
-    public Screen(){}
+public class Screen extends JFrame {
+    // Internal class to render to screen (constraint of JFrame -> JPanel)
+    private MainPannel content = new MainPannel();
+    // Pause, menu, high-schores
+    private Menu menu = new Menu();
+    // Top-level object responsible for game state
+    private Game gameInstance = new Game();
+
+    // JFrame is our top-level container, inside of which we render panes
+    public Screen(){
+        // Not TOTALLY sure how much this does for us...
+        super("Colorado Biking Game");
+        // Settings for window
+        Container contentPane = getContentPane();
+        contentPane.setLayout(new BorderLayout());
+        // Upper pane for time and score and what not
+        JPanel headerPanel = new JPanel();
+        headerPanel.setLayout(new GridLayout(1,5));
+        // Add initial panes
+        contentPane.add(headerPanel, BorderLayout.NORTH);
+        contentPane.add(content, BorderLayout.CENTER);
+
+        setSize(800, 800);
+
+        // Start our gameLoop
+        Thread mainThread = new Thread(content);
+        mainThread.start();
+    }
 
-    // For now we can just run our game through this class
-    // TODO: how to package JARs and where the main() should live
     public static void main(String[] args) {
-        Screen gameInstance = new Screen();
-        gameInstance.setVisible(true);
+        Screen gameScreen = new Screen();
+        gameScreen.setVisible(true);
     }
 
-    // User input affects the game through this method
-    public void actionPerformed(ActionEvent e) {
+    private class MainPannel extends JPanel implements Runnable {
```

Comment:
I would really like to break this out into a separate class... it would help readability a lot.  Really, we need one more class in our class diagram: Window.

If Window was the top-level object, from which screen inherits we could break all the code below this line into its own file.